src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filterMan = people
+    .filter(man => man.sex === 'm')
+    .filter(man => century ? Math.ceil(man.died / 100) === century : true);
+
+  const ageMan = filterMan.filter(man => man.sex === 'm')
+    .map((age) => differenceAge(age));
```

Comment:
Consider renaming the parameter `age` to something like `person` since it represents a person object, not just their age.